sql/moz-fx-data-shared-prod/accounts_backend/nonprod_accounts/view.sql

```diff
@@ -0,0 +1,17 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.accounts_backend.nonprod_accounts`
+AS
+SELECT
+  uid,
+  verifierVersion,
+  verifierSetAt,
```

Comment:
I'd also include `emailVerified`, which is a key signal for whether the account is legit.